### PR TITLE
WeeklyReportSixthWeek

### PR DESCRIPTION
**Pull request by robin-debry**
Thanks it was edited

### CODE REVIEW

File: Documents/Management/WeeklyReports/mainWeeklyReport.md [hunk 1]
```diff
@@ -269,6 +269,27 @@ The challenges encountered this week are still the same as the previous week but
 
 The fifth week was short, the team finished simulating the functionality of each component with LEDs. The team also managed to finish the documentation like the test plan and the user manual. The team is still motivated to finish the project with the presentation next week.
 
+## Week 6 (15-19 April 2024)
+
+The sixth week was for the preparation of the oral on Friday.
```
Comment (MathiasGagnepain) [inline Documents/Management/WeeklyReports/mainWeeklyReport.md:274]: "oral presentation" please cause their is lot of things with "oral" and some of them can be weird.

**Comment by EnzoGuillouche** [comment]
There's a typo in the name of the pull request!

**Review by EnzoGuillouche — APPROVED** [review]
No issues seen, merging approved.

File: Documents/Management/WeeklyReports/weeklyReportSixthWeek.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# SixthWeeklyReport
+
+[Cumulative Weekly Reports](mainWeeklyReport.md)
+
+## Week 6 (15-19 April 2024)
+
+The sixth week was for the preparation of the oral on Friday.
```
Comment (MathiasGagnepain) [inline Documents/Management/WeeklyReports/weeklyReportSixthWeek.md:7]: same as previously please change "oral" by "oral presentation".

**Review by MistzSoftware — CHANGES_REQUESTED** [review]

**Comment by MathiasGagnepain** [comment]
I want to precise before typing "gif oral exam" I just typed "gif oral", please don't do that I saw a lot of thinks I didn't wanted to see. 😰

**Review by MistzSoftware — APPROVED** [review]

**Pull request merged**